WordPress/Classes/ViewRelated/Post/PostEditorAnalyticsSession.swift

```diff
@@ -53,9 +58,11 @@ struct PostEditorAnalyticsSession {
 
     func end(outcome endOutcome: Outcome) {
         let outcome = self.outcome ?? endOutcome
-        let properties = [
-            Property.outcome: outcome.rawValue,
-            ].merging(commonProperties, uniquingKeysWith: { $1 })
+        var properties = [ Property.outcome: outcome.rawValue].merging(commonProperties, uniquingKeysWith: { $1 })
+
+        if let pageTemplateApplied = pageTemplateApplied {
```

Comment:
I'm thinking it might simplify things if we put the `template` property within `commonProperties`, that way we already have the property set for the `apply` event, and we can still override it for the `preview` event.

Reply:
Re-read this comment after my changes and I think it clicked more. You were saying we should include the commonProperties in the apply and preview events too. I can make that change.